osu.Game.Tournament/TournamentSceneManager.cs

```diff
@@ -186,7 +187,7 @@ public void SetScreen(Type screenType)
             var lastScreen = currentScreen;
             currentScreen = target;
 
-            if (currentScreen is IProvideVideo)
+            if (currentScreen.ChildrenOfType<TourneyVideo>().FirstOrDefault()?.VideoAvailable == true)
```

Comment:
A few things make me wince here, namely the `.ChildrenOfType<>()` usage outside of tests, and the fact that the value of `VideoAvailable` is dependent on timing. But it doesn't look like either are going to be huge problem, especially in the latter case where the surrounding code basically guarantees a sync load (see `TournamentSceneManager`'s BDL). So I'm willing to look away, but I'm probably going to let this hang a bit and see if anyone finds any of the above offensive enough to request changes.

Reply:
I initially added a `bool` to the interface but it just felt like way too much implementation (was touching every screen, adding new fields, then mapping to the property) to do something that can be done in a single line. And had more chance of going wrong (screens missing the interface, implementing wrong etc.)

Reply:
`ChildrenOfType` is the lesser of the two mentioned problems for me.

I'm just going to go ahead and merge without spending too much time on this. It's tournament client.